src/Storages/StorageReplicatedMergeTree.cpp

```diff
@@ -8235,9 +8360,9 @@ void StorageReplicatedMergeTree::movePartitionToTable(const StoragePtr & dest_ta
         }
 
         /// Cancel concurrent inserts in range
-        clearLockedBlockNumbersInPartition(*zookeeper, drop_range.partition_id, drop_range.min_block, drop_range.max_block);
+        clearLockedBlockNumbersInPartition(*zookeeper, source_drop_range.partition_id, source_drop_range.min_block, source_drop_range.max_block);
```

Comment:
This requires extra attention. I am using the source drop range, which means the source partition id is in the drop range.

The reasoning behind this decision was: `move partition` does not delete anything on destination, so all drops/deletes will happen on source. Which means source id is the one to be used.